api/middleware/sendData.js

```diff
@@ -0,0 +1,15 @@
+const axios = require('axios');
+const DSURL = process.env.DS_API_URL;
+
+const sendData = (applicationData, role) => {
```

Comment:
Could this be its own middleware? That way the `cacheSignupData` middleware is only handling one thing specifically. We could also pin any response from the DS API to `req`.

Reply:
Converted sendData to it's own middleware.